web_client/.dockerignore

```diff
@@ -1,5 +1,5 @@
-/node_modules
-/build
+node_modules
```

Comment:
Any particular reason for this change ?

Reply:
Even though it's not the case, to me, "/node_modules" reads like a root path on the host system

Reply:
yes but it's not the case and it's less surprising that what you changed it for.

`build` alone means any directory or file named build will be ignored which you will have forgotten by the time you create a `build` directory somewhere in your source, and you'll spend half a day trying to find out about a bug :)